atlasdb-client/src/main/java/com/palantir/atlasdb/schema/stream/StreamTableDefinitionBuilder.java

```diff
@@ -57,8 +58,8 @@ public StreamTableDefinitionBuilder expirationStrategy(ExpirationStrategy expira
         return this;
     }
 
-    public StreamTableDefinitionBuilder hashFirstRowComponent() {
-        hashFirstRowComponent = true;
+    public StreamTableDefinitionBuilder hashFirstNRowComponents(int numberOfComponentsHashed) {
+        this.numberOfComponentsHashed = numberOfComponentsHashed;
```

Comment:
Add a Stream definition with `hashFirstNRowComponents` > 1 to StreamTestSchema or a new schema and test all functionality like ``StreamTest``.

Reply:
Added the tests for a StreamStore in StreamTestSchema.